lms/djangoapps/teams/static/teams/js/views/team_profile.js

```diff
@@ -42,7 +42,8 @@
                         isMember = TeamUtils.isUserMemberOfTeam(memberships, this.context.userInfo.username),
                         isAdminOrStaff = this.context.userInfo.privileged || this.context.userInfo.staff,
                         isInstructorManagedTopic = TeamUtils.isInstructorManagedTopic(this.topic.attributes.type),
-                        maxTeamSize = this.topic.getMaxTeamSize(this.context.courseMaxTeamSize);
+                        maxTeamSize = this.topic.getMaxTeamSize(this.context.courseMaxTeamSize),
+                        showAssignments = !!this.context.teamsAssignmentsUrl;
```

Comment:
nit: could you toss a comment in here, just somehting small like // if teams assignments are not enabled, this url won't be provided ?

Reply:
(so a reader wouldn't have to flip between js and python files)